rust_dev_preview/examples/sending-presigned-requests/src/main.rs

```diff
@@ -150,8 +151,20 @@ async fn send_presigned_request_with_reqwest(
 ) {
     let client = reqwest::Client::new();
     let res = client
-        .request(req.method().clone(), &req.uri().to_string())
-        .headers(req.headers().clone())
+        .request(
+            req.method().try_into().expect("converted method"),
```

Comment:
I would probably use `.parse()` for clarity here
```suggestion
            req.method().parse().expect("converted method"),
```

Reply:
Done